### DISCUSSION TITLE
Is there a way to track user activity via Adaptive cards?

### DISCUSSION THREAD

**Discussion by dejani94** [microsoft/AdaptiveCards#5716] [Q&A]
Hello, I am interested to know is there a way to track user activities as clicks, and trigger Activity to the bot. 
Actually I have data set among the adaptive card, which has multiple urls in it. On the end I want to track which url is clicked and opened. Is there something similar to openUrl action, but with possibility to trigger activity and sent it to the bot?

**Comment by skincarejen19** [comment]
Good question I would love to know also
